### DISCUSSION TITLE
Taskbar window and Widgets overlap

### DISCUSSION THREAD

**Discussion by KamilG5** [zhongyang219/TrafficMonitor#1861] [Q&A]
After Windows 11 update, taskbar Window and Widgets overlap when taskbar is aligned to the left:
![Pasek po lewej](https://github.com/user-attachments/assets/7c082275-dd72-4d4d-bf70-e2fe5c83c51d)

And when I align the bar to the center, the Window taskbar overlaps the applications pinned to the taskbar:
![Pasek na środku](https://github.com/user-attachments/assets/093457eb-8633-4d9c-a29c-360616c16b79)

The only solution is to switch to the main window instead of the taskbar window:
![Main Window](https://github.com/user-attachments/assets/30d2b226-0f7c-44b2-8313-d550e5941ec5)

But maybe there is something you can do to still use the taskbar window

**Comment by DrMirage** [comment]
try a fork

like 

https://github.com/HowerZiu/TrafficMonitor/tree/4hdds

**Comment by KamilG5** [comment]
Thank you for your answer but I solved it by placing the taskbar in the middle and the Window taskbar on the left

![Traffic](https://github.com/user-attachments/assets/d692cfd1-fc75-4b9b-9c39-1bbef3d18032)
